pandas/core/frame.py

```diff
@@ -3135,7 +3135,9 @@ def _setitem_array(self, key, value):
                     key, axis=1, raise_missing=False
                 )[1]
                 self._check_setitem_copy()
-                self.iloc._setitem_with_indexer((slice(None), indexer), value)
+                self.iloc._setitem_with_indexer(
+                    (slice(None), indexer), value, self.iloc.name
```

Comment:
now that the default for "name" is "iloc", you dont need to pass it here

Reply:
Thanks very much. I actually deleted this, but got an error message from Pycharm when running black pandas. Now the message makes sense.